docker/install/ubuntu_install_qemu.sh

```diff
@@ -16,10 +16,32 @@
 # specific language governing permissions and limitations
 # under the License.
 
+#
+# Install QEMU on Ubuntu.
+#
+# Usage: ubuntu_install_qemu.sh [--target-list target0,target1,...]
+#   --target-list is list of target for QEMU comma seperated. e.g. aarch64-softmmu,arm-softmmu,...
+#
+
 set -e
-set -u
 set -o pipefail
 
+# Get number of cores for build
+if [[ "${TVM_CI_NUM_CORES}" ]]; then
```

Comment:
i think you need e.g. `if [ -n "${TVM_CI_NUM_CORES}" ]` or some operator, no?

Reply:
shoot, it was working without `-n`, that's why I missed it. But this is a better syntax.
Thanks for finding this!